beacon-chain/p2p/interfaces.go

```diff
@@ -77,7 +77,7 @@ type PeerManager interface {
 	ENR() *enr.Record
 	DiscoveryAddresses() ([]multiaddr.Multiaddr, error)
 	RefreshENR()
-	FindPeersWithSubnet(ctx context.Context, topic string, index, threshold uint64) (bool, error)
+	FindPeersWithSubnet(ctx context.Context, topic string, index uint64, threshold int) (bool, error)
```

Comment:
what is index? subnet index or validator index? might be better to be more explicit here